### PR TITLE
Opt Out Warning Message and Verification Submitted message styling

### PR DESCRIPTION
**Pull request by ahsan-ul-haq**
Opt Out warning message and verification submitted/successful message styling is changed to look the way in proctoring.
Following are the screenshot after update
screen shot 2015-08-31 at 5 13 12 pm
screen shot 2015-08-31 at 5 14 00 pm

@zubair-arbi @aamir-khan @tasawernawaz 
ECOM-2143
ECOM-2141

### CODE REVIEW

File: edx_reverification_block/xblock/static/html/approved.html [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
   </div>
 </div>
+<div class="faq-reverification-block">
+  <h4>
+    {% trans "See Also" %}
+  </h4>
+  <a class="faq-link">
+    {% trans "Frequently asked questions about verification and earning academic credit" %}
+  </a>
+</div>
```
Comment (zubair-arbi) [inline edx_reverification_block/xblock/static/html/approved.html]: nit: add newline at file end

**Comment by zubair-arbi** [comment]
just a small nit, otherwise lgtm :+1:

**Comment by ahsan-ul-haq** [comment]
@srpearce please review this PR

**Comment by srpearce** [comment]
:+1:

**Pull request merged**